src/common/queries/useAddressMempoolTxsInfinite.ts

```diff
@@ -20,15 +21,19 @@ export function useAddressMempoolTxsInfinite(
   address?: string,
   options: any = {}
 ): UseInfiniteQueryResult<InfiniteData<GenericResponseType<MempoolTransaction>>> {
-  const api = useApi();
+  const apiClient = useApiClient();
   return useInfiniteQuery({
     queryKey: [ADDRESS_MEMPOOL_TXS_INFINITE_QUERY_KEY, address],
-    queryFn: ({ pageParam }: { pageParam: number }) =>
-      api.transactionsApi.getMempoolTransactionList({
-        address,
-        limit: DEFAULT_LIST_LIMIT,
-        offset: pageParam,
-      }),
+    queryFn: async ({ pageParam }: { pageParam: number }) => {
+      if (!address) return undefined;
```

Comment:
I keep seeing this so I won't keep making redundanct commentsm but I would like to know why we are using these checks and also the enabled prop